ios/Handlers/MobileAuthHandler.swift

```diff
@@ -77,15 +76,15 @@ extension MobileAuthHandler: ONGMobileAuthRequestDelegate {
     }
 
     func userClient(_: ONGUserClient, didReceive challenge: ONGPinChallenge, for request: ONGMobileAuthRequest) {
-       //@todo will need this for PUSH PIN?
+       // @todo will need this for PUSH PIN?
```

Comment:
To stay coherent I would change `@todo` into `// FIXME: `, what do you think?

Reply:
Updated and also added the correct ticket numbers there.